### ISSUE TITLE
[one-cmd] Fail 3 tests 

### ISSUE THREAD

**Issue by hseok-oh** [Samsung/ONE#13965]
Fail 3 tests since merging #13897, #13894, #13896

```
one-quantize_022.test FAILED
onecc_043.test FAILED
onecc_044.test FAILED
```

_Originally posted by @hseok-oh in https://github.com/Samsung/ONE/issues/13675#issuecomment-2337505633_

**Comment by hseok-oh** [comment]
`one-quantize_022.log` 

```
2024-09-09 10:08:28.717733: I tensorflow/core/util/port.cc:110] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-09-09 10:08:28.738317: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
/home/nfs/git/ONE/build/out/bin/venv/lib/python3.10/site-packages/tensorflow_addons/utils/tfa_eol_msg.py:23: UserWarning: 

TensorFlow Addons (TFA) has ended development and introduction of new features.
TFA has entered a minimal maintenance and release mode until a planned end of life in May 2024.
Please modify downstream libraries to take dependencies from other repositories in our TensorFlow community (e.g. Keras, Keras-CV, and Keras-NLP). 

For more information see: https://github.com/tensorflow/addons/issues/2807 

  warnings.warn(
/home/nfs/git/ONE/build/out/bin/venv/lib/python3.10/site-packages/tensorflow_addons/utils/tfa_eol_msg.py:23: UserWarning: 

TensorFlow Addons (TFA) has ended development and introduction of new features.
TFA has entered a minimal maintenance and release mode until a planned end of life in May 2024.
Please modify downstream libraries to take dependencies from other repositories in our TensorFlow community (e.g. Keras, Keras-CV, and Keras-NLP). 

For more information see: https://github.com/tensorflow/addons/issues/2807 

  warnings.warn(
WARNING:absl:Found untraced functions such as gen_tensor_dict while saving (showing 1 of 1). These functions will not be directly callable after loading.
usage: one-optimize [-h] [-v] [-V] [-C CONFIG] [-p]
                    [--change_outputs CHANGE_OUTPUTS] [-i INPUT_PATH]
                    [-o OUTPUT_PATH] [--convert_nchw_to_nhwc]
                    [--common_subexpression_elimination]
                    [--expand_broadcast_const] [--nchw_to_nhwc_input_shape]
                    [--nchw_to_nhwc_output_shape] [--fold_add_v2]
                    [--fold_cast] [--fold_densify] [--fold_dequantize]
                    [--fold_dwconv] [--fold_fully_connected] [--fold_gather]
                    [--fold_mul] [--fold_reshape] [--fold_shape]
                    [--fold_sparse_to_dense] [--fold_squeeze]
                    [--forward_reshape_to_unaryop] [--forward_transpose_op]
                    [--fuse_add_to_fullyconnected_bias] [--fuse_add_with_conv]
                    [--fuse_add_with_tconv] [--fuse_add_with_fully_connected]
                    [--fuse_batchnorm_with_conv]
                    [--fuse_batchnorm_with_dwconv]
                    [--fuse_batchnorm_with_tconv]
                    [--fuse_mul_to_fullyconnected_weights]
                    [--fuse_slice_with_tconv] [--fuse_bcq]
                    [--fuse_preactivation_batchnorm] [--fuse_mean_with_mean]
                    [--fuse_mul_with_conv] [--fuse_mul_with_div]
                    [--fuse_mul_with_fullyconnected]
                    [--fuse_transpose_with_mean] [--fuse_horizontal_fc_layers]
                    [--make_batchnorm_gamma_positive]
                    [--fuse_activation_function] [--fuse_instnorm]
                    [--fuse_prelu] [--fuse_gelu] [--fuse_rsqrt]
                    [--replace_cw_mul_add_with_depthwise_conv]
                    [--remove_fakequant] [--remove_gather_guard]
                    [--remove_quantdequant] [--remove_redundant_quantize]
                    [--remove_redundant_reshape]
                    [--remove_redundant_transpose] [--remove_unnecessary_add]
                    [--remove_unnecessary_cast] [--remove_unnecessary_reshape]
                    [--remove_unnecessary_slice]
                    [--remove_unnecessary_strided_slice]
                    [--remove_unnecessary_split]
                    [--remove_unnecessary_transpose]
                    [--replace_non_const_fc_with_batch_matmul]
                    [--replace_sub_with_add] [--replace_with_fc_gelu_fc]
                    [--resolve_customop_add] [--resolve_customop_batchmatmul]
                    [--resolve_customop_matmul]
                    [--resolve_customop_max_pool_with_argmax]
                    [--resolve_customop_splitv]
                    [--shuffle_weight_to_16x1float32]
                    [--substitute_pack_to_reshape] [--substitute_padv2_to_pad]
                    [--substitute_splitv_to_split]
                    [--substitute_squeeze_to_reshape]
                    [--substitute_strided_slice_to_reshape]
                    [--substitute_transpose_to_reshape]
                    [--transform_min_max_to_relu6]
                    [--transform_min_relu_to_relu6]
                    [--transform_sqrt_div_to_rsqrt_mul]
                    [--decompose_hardswish] [--decompose_softmax]
                    [--unroll_unidirseqlstm] [--dynamic_batch_to_single_batch]
one-optimize: error: the following arguments are unrecognized: remove_unnecessary_castremove_unnecessary_reshape
```

`onecc_043.test` and `onecc_044.test` return same error message.

**Comment by hseok-oh** [comment]
@jiwaszki Maybe by https://github.com/Samsung/ONE/pull/13897 ?

**Comment by jiwaszki** [comment]
@hseok-oh thank you for catching and fixing in few minutes!

BTW I just want to understand, are these test run as post-commit only that it wasn't spotted on PRs CI?

**Comment by seanshpark** [comment]
>  thus it wasn't spotted on PRs CI?

Yes, these command tests takes too long to run in PR.

**Comment by hseok-oh** [comment]
Fixed by #13966

**Comment by jiwaszki** [comment]
> > thus it wasn't spotted on PRs CI?
> 
> Yes, these command tests takes too long to run in PR.

I fully understand. I will take greater care about commas then...
